README.md

```diff
@@ -62,56 +69,80 @@ It could be anything from articles to books to videos that describes:
 - [The Uber Go Style Guide](https://github.com/uber-go/guide)
 
 ### Python
+
 - [Hitchhiker's Guide to Python](https://docs.python-guide.org/)
 - [Python API Checklist](https://github.com/vintasoftware/python-api-checklist)
 - [Design Patterns Implementations](https://github.com/faif/python-patterns)
 - [PEP8](https://www.python.org/dev/peps/pep-0008/)
 - [Design Patterns Toptal Blog Post](https://www.toptal.com/python/python-design-patterns)
+- [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html)
 
 ### Ruby
+
 - [RSpec Best Practices](https://github.com/abinoda/rspec-best-practices)
 - [Let’s Read! — Eloquent Ruby](https://medium.com/@baweaver/lets-read-eloquent-ruby-ch-1-b2115d1416a6)
 - [Ruby Style Guide](https://rubystyle.guide/)
 - [Best Ruby](http://franzejr.github.io/best-ruby/)
 
 ### OCaml
+
 - [OCaml Style Guide](https://github.com/lindig/ocaml-style)
 
 ### Erlang
+
 - [Spawned Shelter](http://spawnedshelter.com)
 
 ### Elixir
+
 - [Credo's Elixir Style Guide](https://github.com/rrrene/elixir-style-guide)
 
 ### Common Lisp
+
 - [Standard CL Symbols](https://www.hexstreamsoft.com/articles/notes-tips-standard-common-lisp-symbols/)
 
 ### Java
+
 - [Principles of Lean Java](http://www.ameyalokare.com/software/2018/01/13/lean-java-principles.html)
+- [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html)
 
 ### Scala
+
 - [Scala Idioms](https://leanpub.com/scalaidioms/read)
 - [Scala Best Practice Idioms](https://alvinalexander.com/scala/scala-best-practices-idioms-cookbook)
 
 ### D
+
 - [The D Style](https://dlang.org/dstyle.html)
 - [Pragmatic D Tutorial: Idiomatic D](https://qznc.github.io/d-tut/idiomatic.html)
 
 ### Clojure
-- [Clojure Style Guide](https://github.com/bbatsov/clojure-style-guide)
 
+- [Clojure Style Guide](https://github.com/bbatsov/clojure-style-guide)
 
 ### Nim
+
 - [Nim Style Guide](https://nim-lang.org/docs/nep1.html)
 - [Nim Notes](https://scripter.co/notes/nim)
 
 ### Pony
+
 - [Pony Patterns](https://patterns.ponylang.io/)
 
 ### Javascript
+
 - [Eloquent Javascript](http://eloquentjavascript.net/)
 - [You Don't Know JS](https://github.com/getify/You-Dont-Know-JS)
 - [JS: The Right Way](https://jstherightway.org/)
+- [Google JavaScript Style Guide](https://google.github.io/styleguide/jsguide.html)
+
+### C++
+
+- [Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html)
+
+### General API Design and Tooling
 
-## General API Design and Tooling
 - [Designing unix tools](https://monkey.org/~marius/unix-tools-hints.html)
+
+## Books
+
+- [Clean Code by Robert C Martin](https://www.amazon.in/Clean-Code-Robert-C-Martin/dp/8131773388)
```

Comment:
I would only like to have free resources in this repo. Can you please remove this as this is not free?

Reply:
Updated the PR :)